docs/pages/zero-trust-access/compliance-frameworks/fedramp.mdx

```diff
@@ -28,12 +28,12 @@ Teleport includes FedRAMP and FIPS features to support companies that sell into
 
 |  Control | Teleport Features |
 | - | - |
-| [AC-02 Account Management]((=fedramp.control_url=)AC-02) | Audit events are emitted in the Auth Service when a user is created, updated, deleted, locked, or unlocked. |
+| [AC-02 Account Management]((=fedramp.control_url=)AC-02) | Teleport integrates with SSO providers such as GitHub, Okta, Google, etc. Teleport supports role-based access control (RBAC) for SSH and Kubernetes. Teleport certificate-based SSH and Kubernetes authentication and audit logging comply with these requirements without additional configuration. Audit events are emitted in the Auth Service when a user is created, updated, deleted, locked, or unlocked. |
 | [AC-03 Access Enforcement]((=fedramp.control_url=)AC-03) | Teleport Enterprise supports robust [Role-based Access Controls (RBAC)](../authentication/authentication.mdx) to: <br/>• Control which infrastructure resources a user can or cannot access. <br/>• Control cluster level configuration (session recording, configuration, etc.) <br/>• Control which Unix logins a user is allowed to use when logging into a server. |
 | [AC-07 Unsuccessful Logon Attempts]((=fedramp.control_url=)AC-07) |  Teleport supports two types of users: local and SSO-based accounts (GitHub, Google Apps, Okta, etc). For local accounts, by default, Teleport locks accounts for 30 minutes after 5 failed login attempts. For SSO-based accounts, the number of invalid login attempts and lockout time period is controlled by the SSO provider. |
-| [AC-08 System Use Notification]((=fedramp.control_url=)AC-08) | Teleport integrates with Linux Pluggable Authentication Modules (PAM). PAM modules can be used to display a custom message on login using a message of the day (MOTD) module within the Session management primitive. |
-| [AC-10 Concurrent Session Control]((=fedramp.control_url=)AC-10) | Teleport administrators can define concurrent session limits using Teleport’s RBAC. |
-| [AC-12 Session Termination]((=fedramp.control_url=)AC-12) | Admins can terminate active sessions with [session locking](../../identity-governance/locking.mdx). Teleport terminates sessions on expiry or inactivity. |
+| [AC-08 System Use Notification]((=fedramp.control_url=)AC-08) | Teleport integrates with Linux Pluggable Authentication Modules (PAM). PAM modules can be used to display a custom message on login using a message of the day (MOTD) module within the Session management primitive. Pre-Authentication MOTD is also available. |
+| [AC-10 Concurrent Session Control]((=fedramp.control_url=)AC-10) | Teleport administrators can define concurrent session limits using Teleport's RBAC. Supports both a maximum number of connections (`max_connections`) and the maximum number of simultaneously connected users (`max_users`) under the `connection_limits` configuration parameter. |
```

Comment:
I would find an alternative to naming specific configuration fields here, since it's hard to get much meaning from them without more context than would fit in this table cell.